packages/sampler/stories/default/Layout.js

```diff
@@ -29,7 +29,10 @@ export const _Layout = (args) => (
 			<Cell shrink={args['shrinkable cell']}>
 				<Item>Second</Item>
 			</Cell>
-			<Cell>
+			<Cell
+				grow={args['growable cell']}
+				size={args['cell size'] + 'px'}
```

Comment:
Let's separate the args for the first cell size and this one.

Reply:
I'll fix it.